.storybook/main.ts

```diff
@@ -1,4 +1,5 @@
 import type { StorybookConfig } from '@storybook/web-components-vite';
+import path from 'node:path';
```

Comment:
What's this guy doing?

Reply:
It was from a failed attempt at trying to force the vite config, sorted it with the manual imports anyway